### ISSUE TITLE
[Bug]: Got crashed, possibly Stand

### ISSUE THREAD

**Issue by abriginets** [Yimura/YimMenu#3106]
### Describe the bug

I was delivering a vehicle for exotic exports and got a ton of warnings about some player trying to hijack my car. I've configured Yim to react with kick and block join for everybody who's trying to do that. Apparently that player was able to reconnect to the session though Yim sent me a message with something like "Attempting to block join from ...". After a brief moment I got desync kicked and after automatically reconnecting - crashed. The last thing I saw in the console were messages:

```
[21:59:46.4339569][WARN/notify.cpp:35] Blocked player ped removal crash from Houdini_Playz (235452579)
[21:59:46.4360735][WARN/notify.cpp:35] Blocked invalid weapon type crash from Houdini_Playz (235452579)
```

This looks like something Stand would do. Though some other player tried to crash me with Stand 15-20 minutes before that happend and it was blocked successfully by Yim.

### Steps To Reproduce

1. Get into the session with HoudiniPlayz or AlexRow228 with spoofed host token
2. Get crashed

### Expected Behavior

Do not get crashed

### Logs

Note: stack trace is slightly above the last blocked crash messages

[cout.log](https://github.com/YimMenu/YimMenu/files/15287825/cout.log)

### Store

Steam

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by gir489returns** [comment]
This crashed because you executed the command turn_into_beast. I don't know where exactly in the code because your log is from release (expected), only that it crashed within the function turn_into_beast::execute. This is a manual process.

**Comment by abriginets** [comment]
> EDIT: That was just an exception caught, it looks like it continued execution for several more sections, but died to unknown cause.

Is there a possibility that this exception could lead to a total crash?

> We can't debug it, because there's no crash log from the actual crash.

Let's not close this one yet. I will try joining those players a couple more times and bait them to crash me. Maybe I'll get the stack trace eventually. I will close this issue if there's no luck in acquiring it.

**Comment by Ahmad23145** [comment]
Its the Elegant Crash in Stand

**Comment by Rxann** [comment]
If it is the elegant crash,this should already be fixed?

**Comment by abriginets** [comment]
Like I said I'm not sure. Elegant crash fix was implemented 4 days ago and indeed Yim blocked it for me yesterday
```
[21:28:31.5564483][WARN/notify.cpp:35] Blocked player ped removal crash from IIRxz3ModzII (214785157)
[21:28:31.5584912][WARN/notify.cpp:35] Blocked invalid weapon type crash from IIRxz3ModzII (214785157)
```
Then after a short period of time Yim couldn't block it
```
[21:59:46.4339569][WARN/notify.cpp:35] Blocked player ped removal crash from Houdini_Playz (235452579)
[21:59:46.4360735][WARN/notify.cpp:35] Blocked invalid weapon type crash from Houdini_Playz (235452579)
```

Either Stand already pushed an update on their side, or Yim protection is unreliable (not judging anyone, just listing all the possible options)

**Comment by Rxann** [comment]
> Like I said I'm not sure. Elegant crash fix was implemented 4 days ago and indeed Yim blocked it for me yesterday
> 
> ```
> 
> [21:28:31.5564483][WARN/notify.cpp:35] Blocked player ped removal crash from IIRxz3ModzII (214785157)
> 
> [21:28:31.5584912][WARN/notify.cpp:35] Blocked invalid weapon type crash from IIRxz3ModzII (214785157)
> 
> ```
> 
> Then after a short period of time Yim couldn't block it
> 
> ```
> 
> [21:59:46.4339569][WARN/notify.cpp:35] Blocked player ped removal crash from Houdini_Playz (235452579)
> 
> [21:59:46.4360735][WARN/notify.cpp:35] Blocked invalid weapon type crash from Houdini_Playz (235452579)
> 
> ```
> 
> 
> 
> Either Stand already pushed an update on their side, or Yim protection is unreliable (not judging anyone, just listing all the possible options)

Yes, I do also believe there is a second arg that is being added in a new pr which could explain it.

**Comment by lonelybud** [comment]
> Either Stand already pushed an update on their side, or Yim protection is unreliable (not judging anyone, just listing all the possible options)

their are also crash objects that have not been added to list yet.

**Comment by Rxann** [comment]
> > Either Stand already pushed an update on their side, or Yim protection is unreliable (not judging anyone, just listing all the possible options)
> 
> 
> 
> their are also crash objects that have not been added to list yet. 

Yes, like there is an orgasm crash that I don't think we block afaik and there are more I just don't remember them as vividly.

**Comment by xesdoog** [comment]
> Yes, like there is an orgasm crash that I don't think we block afaik and there are more I just don't remember them as vividly.

*Orgasm crash*? 👀
